$NetBSD: patch-bootstrap,v 1.3 2019/11/27 22:32:27 adam Exp $

Use external libuv; the one included with cmake makes building fail.

--- bootstrap.orig	2020-12-07 15:43:40.025952256 +0000
+++ bootstrap
@@ -220,8 +220,6 @@ if ${cmake_system_mingw}; then
   else
     cmake_default_prefix="c:/Program Files/CMake"
   fi
-elif ${cmake_system_haiku}; then
-  cmake_default_prefix=`finddir B_COMMON_DIRECTORY`
 else
   cmake_default_prefix="/usr/local"
 fi
